js/glue-related.js

```diff
@@ -322,6 +327,48 @@ async function configureNotifications(config) {
   }
 }
 
+async function getNotificationsConfiguration() {
+  const glue = await gluePromise;
+  const { enable, enableToasts } = await glue.notifications.getConfiguration();
```

Comment:
This method is only available on the latest @glue42/desktop, so please make sure it won't break using the older version.

Reply:
Point taken. I've created a new pull request that features this change as well.